src/zope/meta/update_python_support.py

```diff
@@ -107,21 +113,23 @@ def main():
             sys.exit(0)
 
         if no_longer_supported:
-            for version in sorted(list(no_longer_supported)):
-                call(bin_dir / 'addchangelogentry',
-                     f'Drop support for Python {version}.',
-                     *non_interactive_params)
+            version_spec = ', '.join(sorted(no_longer_supported))
+            call(bin_dir / 'addchangelogentry',
+                 f'Drop support for Python {version_spec}.',
+                 *non_interactive_params)
             python_versions_args.append(
                 '--drop=' + ','.join(no_longer_supported))
 
         if not_yet_supported:
-            for version in sorted(list(not_yet_supported)):
-                call(
-                    bin_dir / 'addchangelogentry',
-                    f'Add support for Python {version}.',
-                    *non_interactive_params)
-            python_versions_args = ['--add=' +
-                                    ','.join(supported_python_versions())]
+            version_spec = ', '.join(sorted(not_yet_supported))
```

Comment:
Does this properly sort 3.9 before 3.10?

(I suppose it doesn't matter; I don't think there are any packages left where we would be _adding_ 3.9 support.)

Reply:
Not it does not properly sort and it did not properly sort before my changes. I (and probably Michael as the original author as well) didn't think it worthwhile to add the complication of parsing all these strings into proper version specifications, sorting them and then turning them back into strings just so they are sorted nicer in a change log entry. That is the only place where you would see it.